README.md

```diff
@@ -6,12 +6,13 @@ Dependency
 
 * C++ primitiv
 * Python 3 (3.5 or later)
+* NumPy
```

Comment:
Which version is supported?

Reply:
I couldn't find the actual minimum required version, so I added the version that contained in Ubuntu 16.04.